.github/workflows/ci.yml

```
@@ -50,31 +52,31 @@ jobs:
           - stable-x86_64-apple-darwin
           - stable-x86_64-unknown-linux-gnu
         include:
-          - { toolchain: 1.70.0-x86_64-pc-windows-msvc  , os: windows-2019 }
-          - { toolchain: 1.70.0-x86_64-apple-darwin     , os: macos-11     }
-          - { toolchain: 1.70.0-x86_64-unknown-linux-gnu, os: ubuntu-20.04 }
-          - { toolchain: stable-x86_64-pc-windows-msvc  , os: windows-2019 }
-          - { toolchain: stable-x86_64-apple-darwin     , os: macos-11     }
-          - { toolchain: stable-x86_64-unknown-linux-gnu, os: ubuntu-20.04 }
+          - { toolchain: 1.70.0-x86_64-pc-windows-msvc, os: windows-2022 }
+          - { toolchain: 1.70.0-x86_64-apple-darwin, os: macos-14 }
+          - { toolchain: 1.70.0-x86_64-unknown-linux-gnu, os: ubuntu-24.04 }
+          - { toolchain: stable-x86_64-pc-windows-msvc, os: windows-2022 }
+          - { toolchain: stable-x86_64-apple-darwin, os: macos-14 }
+          - { toolchain: stable-x86_64-unknown-linux-gnu, os: ubuntu-24.04 }
```

For macOS, I guess AArch64 is more popular now. Or rather, [`macos-14` is Apple silicon one](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#choosing-github-hosted-runners). So:

```suggestion
          - { toolchain: 1.70.0-x86_64-pc-windows-msvc, os: windows-2022 }
          - { toolchain: 1.70.0-aarch64-apple-darwin, os: macos-14 }
          - { toolchain: 1.70.0-x86_64-unknown-linux-gnu, os: ubuntu-24.04 }
          - { toolchain: stable-x86_64-pc-windows-msvc, os: windows-2022 }
          - { toolchain: stable-aarch64-apple-darwin, os: macos-14 }
          - { toolchain: stable-x86_64-unknown-linux-gnu, os: ubuntu-24.04 }
```

---

Good catch! Fixed